services/horizon/internal/actions_offer.go

```diff
@@ -50,6 +50,11 @@ func (action *OffersByAccountAction) SSE(stream sse.Stream) {
 	)
 }
 
+// GetTopic is a method for actions.SSE
+func (action *OffersByAccountAction) GetTopic() string {
+	return action.GetString("account_id")
```

Comment:
are you sure the arg her eshould be `account_id`? it's already used in `func (action *DataShowAction) GetTopic()`